### DISCUSSION TITLE
persp-mode or perspective-el

### DISCUSSION THREAD

**Discussion by npajkovsky** [abougouffa/minemacs#176] [Ideas]
Hello,

do you have any plans to add persp-mode or perspective-el? It's one of the most important feature that I use in Doom emacs, which you don't have, and kind of blocks me to moving to your project.

**Comment by abougouffa** [comment]
Hi @npajkovsky 

Nop, I'm not a fan of the way persp-mode and perspective-el work. I'm adopting a more minimalist way of managing workspaces using my [one-tab-per-project](https://github.com/abougouffa/one-tab-per-project), so no plans to integrate other packages.
